src/Symfony/Component/HttpFoundation/AcceptHeader.php

```diff
@@ -78,23 +76,19 @@ public function __toString()
     /**
      * Tests if header has given value.
      *
-     * @param string $value
-     *
      * @return bool
      */
-    public function has($value)
+    public function has(string $value)
     {
         return isset($this->items[$value]);
     }
 
     /**
      * Returns given value's item, if exists.
      *
-     * @param string $value
-     *
      * @return AcceptHeaderItem|null
      */
-    public function get($value)
+    public function get(string $value)
```

Comment:
Should we use this code?

```php
public function get(string $value) ?AcceptHeaderItem
```

Reply:
👎  Introducing a return type to a non-final method is a breaking change.

Reply:
This PR is targeted to master branch, I think breaking changes are valid if they are really useful. It's just my question for more details.

Reply:
In fact, I miss the case "Don't add return types. They require a different strategy.".